docs/source/environments.rst

```diff
@@ -26,11 +26,13 @@ Packaged Environments
 
 The package comes with a few *default environments* which are **always available** and designed for specific schedulers.
 That includes the :py:class:`~flow.DefaultTorqueEnvironment` and the :py:class:`~flow.DefaultSlurmEnvironment`.
-This means that if you are within an environment with a *torque* or *slurm scheduler* you should be immediately able to submit to the cluster.
+This means that if you are within an environment with a *Torque* or *Slurm scheduler* you should be immediately able to submit to the cluster.
 
 In addition, **signac-flow** comes with some environments tailored to specific compute clusters that are defined in the :py:mod:`flow.environments` module.
-These environments are not automatically available.
-Instead, you need to *explictly import* the :py:mod:`flow.environments` module.
+These environments are also automatically available, but if they conflict with a specific environment of your choice, you can opt out of using these environments by setting the ``flow.IMPORT_ENVIRONMENTS`` signac config variable.
```

Comment:
Please rename "IMPORT_ENVIRONMENTS" to "auto_import_environments" similar to my change request in the corresponding PR.